### ISSUE TITLE
False positive when pattern matching functions parameters with sigils

### ISSUE THREAD

**Issue by tuvistavie** [rrrene/credo#63]
The following code gives an incorrect warning:

```elixir
defmodule Foobarbaz do
  def foo(~w(a b c)) do
    IO.puts("a b c")
  end
  def foo(_) do
    IO.puts("not a b c")
  end
end
```

warning:

```
┃ [W] ↗ Parameter `sigil_w` has same name as the `Kernel.sigil_w` macro.
┃       lib/foobarbaz.ex:2 (Foobarbaz.foo)
```

Thank you for your help!

**Comment by rrrene** [comment]
Interesting. Thx for reporting! :+1:
